ea-factories/src/factories/balance.ts

```diff
@@ -0,0 +1,120 @@
+import {
+  ExecuteFactory,
+  Config,
+  DataResponse,
+  Account,
+  SequenceResponseData,
+} from '@chainlink/types'
+import objectPath from 'object-path'
+import { Validator, AdapterError } from '@chainlink/external-adapter'
+import { util } from '@chainlink/ea-bootstrap'
+
+const DEFAULT_DATA_PATH = 'addresses'
+const DEFAULT_CONFIRMATIONS = 6
+
+const WARNING_UNSUPPORTED_PARAMS = 'No Operation: this provider does not support'
+const ERROR_MISSING_ADDRESS = 'No Operation: address param is missing'
+
+export function addresses(accounts: Account[]) {
+  return accounts.reduce<string[]>((accumulator, current) => {
```

Comment:
We are just filtering accounts without a warning and mapping them to address, no need to do a more complex sequence reduction.

I also think that this function should not be exported. I see it used in only one place and we can instead do `accounts.map(a => a.address)` directly, when needed.